pkg/config/config.go

```diff
@@ -774,12 +786,13 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 
 	// check for validation on execution
 	if onExecution {
-		if err := c.ValidateRuntimes(); err != nil {
+		var err error
+		if err = c.ValidateRuntimes(); err != nil {
```

Comment:
I don't think this is an improvement. Same for similar code below. Can you drop this?

Reply:
fixed